OMX-27-firmware/OMX-27-firmware.ino

```diff
@@ -199,7 +201,7 @@ void resetClocks(){
 
 void setGlobalSwing(int swng_amt){
 	for(int z=0; z<NUM_PATTERNS; z++) {
```

Comment:
This pattern seems to be used a lot, you could [use a macro like this](https://github.com/winder/TSynth-Teensy4.1/blob/df4e86ec17db07a5ce09338eb3333bc8e48762e1/TSynth/VoiceGroup.cpp#L5):
```
#define FOR_EACH_PATTERN(CMD)             \
    for(int z=0; z<NUM_PATTERNS; z++) {   \
    {                                     \
        sequencer.getPattern(z)->CMD;     \
    }
```

Also maybe move anything that needs it into the sequence class?

Reply:
Could also implement the a `forEachPattern` method on Sequencer that accepted a lambda maybe?